### PR TITLE
Added SettingsScreen.

### PR DESCRIPTION
**Pull request by RomanEsin**

### CODE REVIEW

File: Project SF/Views/Tabs/Settings/SettingsView.swift [hunk 1]
```diff
@@ -0,0 +1,63 @@
+//
+//  SettingsView.swift
+//  Project SF
+//
+//  Created by Roman Esin on 11.07.2020.
+//
+
+import SwiftUI
+
+struct SettingsLabel<Destination: View>: View {
```
Comment (wfltaylor) [inline Project SF/Views/Tabs/Settings/SettingsView.swift]: Maybe rename to be more general (ie "NavigationLabel") and move to its own file

Comment (RomanEsin) [reply Project SF/Views/Tabs/Settings/SettingsView.swift]: Yep, just gotta fix some conflicts

**Comment by RomanEsin** [comment]
@wfltaylor

**Comment by wfltaylor** [comment]
HealthKitController seems to regress for some reason

**Comment by RomanEsin** [comment]
wdym?

**Review by wfltaylor — CHANGES_REQUESTED** [review]
HealthKitController seems to have lost lots of code

**Comment by RomanEsin** [comment]
Ok, will clean it up in the next PR

**Pull request merged**